code/src/index.css

```diff
@@ -1,8 +1,25 @@
+:root {
+  --color-primary: #6971f8;
+  --color-primaryFaded: #b4b8fc;
+  --color-greyFaded: #f4f4f4;
+  --color-grey: #8f91a0;
+  --color-black: #383230;
+  --color-white: #fbfbfd;
+}
```

Comment:
Great to declare all colors in root, I should start to do that as well. Give s great structure :)